cfn-guard/src/parser.rs

```diff
@@ -156,6 +156,8 @@ fn destructure_rule(rule_text: &str, cfn_resources: &HashMap<String, Value>) ->
                     match &caps["operator"] {
                         "==" => OpCode::Require,
                         "!=" => OpCode::RequireNot,
+                        "<=" => OpCode::SmallerAs,
```

Comment:
This is a great start!  I think we need to smooth out the ergonomics of the operators a bit, though.  

I think there are two subtle but important changes:
1.  The numeric operator should be on the left side of the `=` in both cases or it starts to look like an arrow, and
2.  The rule operator looks like it's saying "lesser/greater than or equal to" but the actual check is doesn't include equality.  I think we should expand this to two more operators & operations:

< Less Than
> Greater Than
<= Less than or equal to
>= Greater than or equal to